.github/workflows/build.yaml

```diff
@@ -54,7 +54,7 @@ jobs:
       
       - uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 8.x
```

Comment:
May be better to just leave this off and use whatever version is default (usually latest lts iirc)

Reply:
https://github.com/hk-modding/api/blob/818e49fc6a0682808172b9f1f3beb9af4d6f591c/.github/workflows/build.yaml#L55-L57 does the following already in the actions: <https://github.com/hk-modding/api/actions/runs/16009882284/job/45165073689#step:6:14> & <https://github.com/hk-modding/api/actions/runs/16009882284/job/45165073689#step:6:25>.

(it already installs both 8.x and 6.0.x)